.jshintrc

```diff
@@ -0,0 +1,17 @@
+{
+  "browser": false,
+  "node": true,
+  "globals": {
+    "require": true
+  },
+  "predef": {
+  },
+  "eqeqeq": true,
+  "quotmark": true,
+  "smarttabs": true,
+  "trailing": true,
```

Comment:
What do `smarttabs` and `trailing` options do? I couldn't find them in the jshint docs.